lib/Bio/DB/HTS.pm

```diff
@@ -1346,7 +1346,7 @@ sub hts_file { shift->{hts_file} }
 sub is_remote {
     my $self = shift;
     my $path = shift;
-    return $path =~ /^(http|https|ftp):/;
+    return $path =~ /^(http|https|ftp|s3):/;
```

Comment:
More of a comment about the original method. This is a boolean method that returns `1` or a blank string. That's very odd. Would be better to have it return `1` or `0` IMO